backend/worker/kafka_worker.go

```diff
@@ -126,8 +126,25 @@ type KafkaWorker struct {
 	WorkerThread int
 }
 
+func (k *KafkaBatchWorker) log(ctx context.Context, fields log.Fields, msg ...interface{}) {
+	if k.lastPartitionId != nil {
+		return
+	}
+
+	partitionId := *k.lastPartitionId
+	if partitionId == 408 || partitionId == 58 || partitionId == 289 {
```

Comment:
I added these to match `KafkaWorker.log`, but not sure if these should be different or more?

Reply:
batched topics only have 100 partitions, so we should do something based on that
```suggestion
	if partitionId % 25 == 0 {
```